### PR TITLE
Domashka 5

### PR DESCRIPTION
**Pull request by MiholenkoAndrei**
DONE

*The hard folder is not completely done
*Folders easy and medium are made completely

### CODE REVIEW

File: tasks/easy/if_elif_else/is_divide_by.py [hunk 1]
```diff
@@ -36,9 +36,10 @@ def is_divide_by(to_divide: int, divider_1: int, divider_2: int) -> bool:
     :return: True, если делится на 2 числа, False, если нет
     :rtype: bool
     """
-    result = None
-    # TODO написать свой код здесь
-    return result
+    if to_divide % divider_1 is to_divide % divider_2:
```
Comment (rineisky) [inline tasks/easy/if_elif_else/is_divide_by.py]: and

File: tasks/easy/if_elif_else/square_or_rectangle.py [hunk 2]
```diff
@@ -13,8 +13,10 @@
 
 
 def square_or_rectangle(side1: int, side2: int) -> int:
-    result = None
-    return result
+    if side1 * side1 == side1 * 2:
```
Comment (rineisky) [inline tasks/easy/if_elif_else/square_or_rectangle.py]: if side1 == side 2: else:

**Review by rineisky — APPROVED** [review]
well done

File: tasks/easy/if_elif_else/is_divide_by.py (hunk 1 above)
Comment (MiholenkoAndrei) [reply tasks/easy/if_elif_else/is_divide_by.py]: updated

File: tasks/easy/if_elif_else/square_or_rectangle.py (hunk 2 above)
Comment (MiholenkoAndrei) [reply tasks/easy/if_elif_else/square_or_rectangle.py]: updated

**Pull request merged**